content/statefulset/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Stateful containers using StatefulSets"
+chapter: true
+weight: 50
```

Comment:
I think the weight should be changed to order this chapter after the helm chapter.

Reply:
helm was 62 so I changed the weight to 63.